composer.json

```diff
@@ -1,17 +1,17 @@
 {
-  "name": "cloudflare/sdk",
+  "name": "smartframe-technologies/cloudflare-php",
   "description": "PHP binding for v4 of the Cloudflare Client API.",
   "type": "library",
   "require": {
-    "guzzlehttp/guzzle": "^7.0.1",
-    "php": ">=7.2.5",
+    "guzzlehttp/guzzle": "*",
```

Comment:
I think that you should stick to the current major version cause it could cause problems in the future when another major version would be published

Reply:
done